File: app/src/main/java/com/ivanmagda/habito/barchart/formatters/HabitoBarChartValueFormatter.java
package com.ivanmagda.habito.barchart.formatters;

import com.github.mikephil.charting.data.Entry;
import com.github.mikephil.charting.formatter.IValueFormatter;
import com.github.mikephil.charting.utils.ViewPortHandler;

public final class HabitoBarChartValueFormatter implements IValueFormatter {

    @Override
    public String getFormattedValue(float value, Entry entry, int dataSetIndex, ViewPortHandler viewPortHandler) {
        return String.valueOf((int) value);
    }

}

File: app/src/main/java/com/ivanmagda/habito/barchart/formatters/HabitoBarChartValueFormatter.kt
package com.ivanmagda.habito.barchart.formatters

import com.github.mikephil.charting.data.Entry
import com.github.mikephil.charting.formatter.IValueFormatter
import com.github.mikephil.charting.utils.ViewPortHandler

class HabitoBarChartValueFormatter : IValueFormatter {
    override fun getFormattedValue(value: Float, entry: Entry, dataSetIndex: Int, viewPortHandler: ViewPortHandler): String {
        return value.toInt().toString()
    }
}
